src/ui/components/WalletConnectRequest/WalletConnectRequest.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from "react";
```

Comment:
Please see previous comments and refactor this component to be passed as a child to the side slider component, which will be used as a child of the "Menu" tab.